src/main/java/ch/njol/skript/sections/SecWhile.java

```diff
@@ -126,4 +240,31 @@ public void exit(Event event) {
 		super.exit(event);
 	}
 
+	private Conditional<Event> getConditional() {
+		return conditional;
+	}
+
+	private boolean isDoWhile() {
+		return doWhile;
+	}
+
+	private @Nullable Node getNextNode(Node precedingNode, ParserInstance parser) {
```

Comment:
This appears to be able to be a static method

Reply:
I was actually working on somewhat adding this to `TriggerItems` 
or another class if you suggest

Reply:
You copied it from SecConditional right? Maybe it could go on the Node class? I'm not 100% sure.